src/styles/utils/_keyframes.scss

```diff
@@ -0,0 +1,9 @@
+@keyframes move {
```

Comment:
Add a comment above the keyframes definition explaining the purpose of the animation and any additional details if needed 📝